src/server/static/js/lines.js

```diff
@@ -276,32 +279,30 @@ function updatePlot(opt) {
   // Load and preprocess the per-region graph data
   loadGleamvizTraces(baseData.regions[selected.region], function (mitigTraces, maxVal) {
     layout.yaxis.range = [0, maxVal];
-    // redraw the lines on the graph
     AddCriticalCareTrace(mitigTraces[selected.mitigation]);
+    // redraw the lines on the graph
     Plotly.newPlot(plotyGraph, mitigTraces[selected.mitigation], layout, plotlyConfig);
   });
 }
 
 function AddCriticalCareTrace(traces) {
+  // add the line only if it doesn't exist yet
+  const lastTrace = traces[traces.length - 1];
+  if (lastTrace && lastTrace.name === "Hospital critical care capacity") return;
```

Comment:
Thanks, I know that mine addition was a bit a poorsman solution, I am still picking up on JS... :see_no_evil: